src/utilities/omit.js

```diff
@@ -0,0 +1,5 @@
+export default function omit(object = {}, ...keys) {
+  const copy = Object.assign({}, object);
+  keys.forEach(key => delete copy[key]);
+  return copy;
+}
```

Comment:
> delete is direct deoptimization for v8

Please explain :D

> as far as you do copy on your own. its better to create fresh copy and then add every non-omitted key-val into copy from original object.

That would the time complexity O(objSize * keysLength) instead of O(objSize + keysLength) assuming that delete is constant but my assumption may be wrong due to your prevoius comment that I do not fully grasp

> for the record Object.assign is shallow operation, so its not true copy. but its okay in this case

Yes, might have been a poor choice of variable naming. My intention was not to do a deep copy.